Packs/DeveloperTools/Scripts/GetInstanceName/GetInstanceName.py

```diff
@@ -1,32 +1,46 @@
 import demistomock as demisto
 from CommonServerPython import *
 from CommonServerUserPython import *
-from typing import Dict, Any, Tuple
+from typing import Dict, Any, List
 
 
-def instance_check(instances, integration_name: str) -> Tuple[bool, Any]:
+def instance_check(instances, integration_name: str) -> List:
+    instance_names = []
     for instance_name, details in instances.items():
         if details.get('brand') == integration_name:
-            return True, instance_name
-    return False, None
+            instance_names.append(instance_name)
+
+    return instance_names
 
 
 def get_instance_name_command(args: Dict[str, Any]) -> CommandResults:
     integration_name = args.get('integration_name', '')
 
     instances = demisto.getModules()
 
-    found, instance_name = instance_check(instances, integration_name)
+    instance_names = instance_check(instances, integration_name)
 
-    if not found:
+    if not instance_names:
         raise DemistoException(f'No instance for integration {integration_name}.')
 
+    if argToBoolean(args.get('return_all_instances')):
+        return CommandResults(
+            outputs_prefix='Instances',
+            outputs_key_field='',
```

Comment:
In case the user runs this on two different integrations you don't want to overwrite the data
```suggestion
            outputs_key_field='integration_name',
```

Reply:
This will override itself in case of multiple values for the same integration name

Reply:
This is the current implementation:
![image](https://user-images.githubusercontent.com/30797606/185444634-b815cab1-b325-4192-a52e-88918db3eeb3.png)

Reply:
:shtak: